src/subscription_manager/repofile.py

```diff
@@ -152,6 +155,9 @@ def from_ent_cert_content(cls, content, baseurl, ca_cert, release_source):
         repo['sslcacert'] = ca_cert
         repo['metadata_expire'] = content.metadata_expire
 
+        if HAS_DEB822 is True:
+            repo['arches'] = content.arches
```

Comment:
Why do you set the `repo['arches']` here? You delete it on line 482. Why isn't `repo['Architectures']` set directly? Maybe it has some reason, but I cannot see this reason.